clients/scheduler/src/hooks/app-context.tsx

```diff
@@ -121,7 +128,10 @@ export const ContextWrapper = ({ children, values }: ContextWrapperProps) => {
   const createAppointment = useCallback(() => {
     postAppointment({
       setScreen: () => setScreen('CONFIRM'),
-      setError,
+      setError: (msg) => { 
+        values.callbacks.onBookingError(msg); 
+        return setError(msg) 
```

Comment:
This is more of a nit than my last comment:

I don't think we should `return` here. I don't think it is adding any value.